### PR TITLE
[ADD] account_find_balance_inconsistency PoC

### PR DESCRIPTION
**Pull request by hbrunn**
This module helps you find inconsistencies in your balance sheet.

It will generate (by default) the asset and liability reports for a given
period, and stop at the first date those two yeidl different results. Then
it takes the user to the accounting items from this day, where corrective
action can be taken. When done, the wizard can be restarted until all issues
are resolved.

### CODE REVIEW

File: account_find_balance_inconsistency/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
```
Comment (NL66278) [inline account_find_balance_inconsistency/__init__.py]: No more copyright in init files.

File: account_find_balance_inconsistency/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Find balance inconsistencies",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "",
```
Comment (NL66278) [inline account_find_balance_inconsistency/__manifest__.py]: category Accounting

File: account_find_balance_inconsistency/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This module helps you find inconsistencies in your balance sheet.
+
+It will generate (by default) the asset and liability reports for a given
+period, and stop at the first date those two yeidl different results. Then
```
Comment (NL66278) [inline account_find_balance_inconsistency/readme/DESCRIPTION.rst]: spelling error in yield

File: account_find_balance_inconsistency/tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
```
Comment (NL66278) [inline account_find_balance_inconsistency/tests/__init__.py]: Delete this line also

File: account_find_balance_inconsistency/tests/test_account_find_balance_inconsistency.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
```
Comment (NL66278) [inline account_find_balance_inconsistency/tests/test_account_find_balance_inconsistency.py]: from odoo...

File: account_find_balance_inconsistency/wizards/__init__.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
```
Comment (NL66278) [inline account_find_balance_inconsistency/wizards/__init__.py]: and this

File: account_find_balance_inconsistency/wizards/account_find_balance_inconsistency.py [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+                ]
+                return {
+                    'type': 'ir.actions.act_window',
+                    'name': _('Journal items on first day of inconsistency'),
+                    'res_model': 'account.move.line',
+                    'views': [(False, 'tree'), (False, 'form')],
+                    'domain': domain,
+                }
+            date += timedelta(days=1)
+        raise exceptions.UserError(_(
```
Comment (NL66278) [inline account_find_balance_inconsistency/wizards/account_find_balance_inconsistency.py:75]: It is a little bit ironic to have no inconsistencies as an exception ;-)

**Comment by hbrunn** [comment]
@NL66278 thanks for the review, also added a test

**Pull request closed without merging**